ui/scripts/system.js

```diff
@@ -9599,6 +9600,23 @@
                             },
                             detailView: {
                                 name: 'label.virtual.appliance.details',
+                                viewAll: [{
+                                    label: 'label.account',
+                                    path: 'accounts',
+                                    preFilter: function(args) {
+                                        if (args.context.routers[0].projectid)
+                                            return false;
+                                        if (args.context.routers[0].account == 'system')
```

Comment:
Is it for System VMs? If so, then it won't appear on this page.

Reply:
For shared networks, VR's account owner is 'system' we don't want to show 'View account' otherwise it fails as you cannot listAccounts for account=system.